schemas/rpglint.json

```diff
@@ -85,6 +85,11 @@
 			],
 			"description": "Force which directive which must be used to include other source. (Copy or Include)"
 		},
+		"LowercaseDirectives": {
```

Comment:
As mentioned in my reply, let's not have `LowercaseDirectives`.

Reply:
Yep, I've taken this out in favour of `DirectiveCasing`.